config.lua

```diff
@@ -14,6 +14,9 @@ PacifistMod.military_entity_types = {
     "electric-turret",
     "fluid-turret",
     "artillery-wagon",
+    "unit",
```

Comment:
We'll need a follow-up issue to make mods compatible that use units. Mining Drones and all its forks comes to mind, also Transport Drones, possibly other similar mods. I would guess that Pyanodon's has a few units as well.

Others that I found with a quick search: 
[ERM - Terran Units for Players](https://mods.factorio.com/mod/erm_terran)
[ERM - Zerg Units / Jormungand Brood](https://mods.factorio.com/mod/erm_zerg_exp)
[ERM - Protoss Units / Velari Tribe](https://mods.factorio.com/mod/erm_toss_exp)
[Total Automization](https://mods.factorio.com/mod/Total_Automization) (plus forks)